salt/states/pkg.py

```diff
@@ -1241,10 +1245,14 @@ def installed(
     failed_hold = None
     if targets or to_reinstall:
         reinstall = bool(to_reinstall)
+        force = False
+        if salt.utils.is_freebsd():
+            force = True    # Downgrades need to be forced.
```

Comment:
@rickyninja sure thing! It'll work as-is for now, but the code should be moved into pkgng.py probably. We try to keep platform-specific stuff out of the state module when possible, because it results in different behavior when executed via the state and via just calling ``pkg.install`` on the salt CLI.